src/store/substrate/event-types.json

```diff
@@ -116,63 +12,56 @@
       },
       "orders": {
         "OrderPaid": {
-          "message": "",
           "value": "id",
           "value_message": "A new order",
           "route": "lab-dashboard-process-order",
           "params": "orderId",
           "identity": "sellerId"
         },
         "OrderRefunded": {
-          "message": "You’ve received ",
           "value": "id",
-          "value_message": "as quality control fees for",
+          "value_message": "You’ve received [] as quality control fees for",
```

Comment:
Why is it like this?

Reply:
as per discuss with the team FE about notification.

`- FE: standarisasi pemakaian event types
    - cek di polkadot provider event types, mana aja yg kepake dan ngga
    - ditaro di value_message semua wordingnya, dengan identifier [] sebagai parameter. contoh : Congrats! You’ve got [] DBIO as a reward for completing the request test from the service requested`